src/makeInfinityAdder.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+  const adder = function(num) {
+    // якщо if (!num), то не проходить тест
+    // Should work with chains containing zero as argument
+
+    if (arguments.length === 0) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    } else {
+      sum += num;
```

Comment:
```suggestion
      return result;
    }
    sum += num;
```
else можна не використовувати, якщо попередня перевірка містить return